test/unit/src/common/actions/t_register-name.js

```diff
@@ -198,10 +201,9 @@ describe('register name actions', () => {
 
         beforeEach(() => {
           // XXX check Authorization header
-          scope
-            .post('/api/store/register-name', { snap_name: 'test-snap' })
+          storeScope = nock(conf.get('STORE_API_URL'))
```

Comment:
Well... yes, but I wanted to avoid a large renaming (to keep scope of PR clean). That said, this :eyes: fairly isolated and small, will try to squeeze it in.